shpc/main/modules/templates/singularity.tcl

```diff
@@ -74,6 +74,11 @@ set inspectCmd "singularity \${SINGULARITY_OPTS} inspect \${SINGULARITY_COMMAND_
 # set_shell_function takes bashStr and cshStr
 set-alias {|module_name|}-shell "${shellCmd}"
 
+{% if wrapper_scripts %}
+# add path with "alias" wrapper scripts to PATH
+{% if aliases %}prepend-path PATH "{{ wrapper_dir }}"{% endif %}
+
```

Comment:
I think this is going to create a lot of empty space - if possible let's clean out the extra newlines.

Reply:
addressed in https://github.com/marcodelapierre/singularity-hpc/commit/6466461228b9c2d1d6afd62ace444c863ee3e98f

Reply:
also tested